.travis.yml

```diff
@@ -13,19 +13,38 @@ stages:
   - test # Default stage with job matrix
   - osx
 
-compiler:
-  - gcc
-  - clang
+# compiler:
+#   - gcc
+#   - clang
 
-env:
-  global:
-    - TRAVIS_CMAKE_GENERATOR="Unix Makefiles"
-  matrix:
-    - TRAVIS_BUILD_TYPE="Release" UBUNTU="xenial", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
-    - TRAVIS_BUILD_TYPE="Debug" UBUNTU="xenial", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
-    - TRAVIS_BUILD_TYPE="Release" UBUNTU="bionic", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
-    - TRAVIS_BUILD_TYPE="Debug" UBUNTU="bionic", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
+# env:
+#   global:
+#     - TRAVIS_CMAKE_GENERATOR="Unix Makefiles"
+#   matrix:
+#     - TRAVIS_BUILD_TYPE="Release" UBUNTU="xenial", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
+#     - TRAVIS_BUILD_TYPE="Debug" UBUNTU="xenial", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
+#     - TRAVIS_BUILD_TYPE="Release" UBUNTU="bionic", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
+#     - TRAVIS_BUILD_TYPE="Debug" UBUNTU="bionic", ROBOTOLOGY_ENABLE_DYNAMICS=TRUE
 
+addons:
+  homebrew:
+    packages:
+    - ccache
+    - ace
+    - asio
+    - eigen
+    - boost
+    - tinyxml
+    - swig
+    - qt5
+    - gsl
+    - pkg-config
+    - jpeg
+    - sqlite
+    - tinyxml
+    - gazebo9
```

Comment:
I suspect including `gazebo9` here will create you problems, as `gazebo9` is installed by the custom tap `osrf/simulation`. As for sure `gazebo9` will never be installed by default in the Travis image, I think it is ok to install it via `brew install gazebo9`, **after** the `brew tap osrf/simulation` (or you can also install directly with `brew install osrf/simulation/gazebo9` .

Reply:
yess, I see. I was just testing the addons.

> you can also install directly with `brew install osrf/simulation/gazebo9`

The homebrew addons should support also `tap`, so we may be able to do it directly there.
Something like:
```
addons:
  homebrew:
    taps:
        - osrf/simulation
    packages:
        - gazebo9
```

Reply:
Great!